src/app/components/DataLoader/DataLoader.jsx

```diff
@@ -1,10 +1,73 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+import Actions from '@Actions';
+import { ajaxCall } from '@utils';
+import appConfig from '@appConfig';
 
 class DataLoader extends React.Component {
+  constructor(props) {
+    super(props);
+    const { location } = this.props;
+    this.pathInstructions = [
+      {
+        expression: /\/research\/collections\/shared-collection-catalog\/bib\/(b\d*)/,
+        pathType: 'bib',
+      },
+    ];
+    this.pathType = null;
+    this.routes = {
+      bib: {
+        apiRoute: () => `${appConfig.baseUrl}/api/bib?bibId=${this.matchData[1]}`,
+        actions: [Actions.updateBib],
+        errorMessage: 'Error attempting to make an ajax request to fetch a bib record from ResultsList',
+      },
+    };
+    this.reducePathExpressions = this.reducePathExpressions.bind(this);
+  }
+
+  componentDidMount() {
+    // Needs to add trackdiscovery actions
+    window.dataLoaderLocation = window.dataLoaderLocation || [];
+    window.dataLoaderLocation.push(this.props.location);
```

Comment:
Add comments to explain why this is here, even if it's for debugging.

Reply:
Removed